include/h2o.h

```diff
@@ -321,6 +322,13 @@ typedef struct st_h2o_status_handler_t {
 } h2o_status_handler_t;
 
 typedef H2O_VECTOR(h2o_status_handler_t) h2o_status_callbacks_t;
+
+typedef enum h2o_forward_informational {
+    H2O_FORWARD_INFORMATIONAL_NONE,
+    H2O_FORWARD_INFORMATIONAL_EXCEPT_H1,
+    H2O_FORWARD_INFORMATIONAL_ALL
+} h2o_forward_informational_t;
```

Comment:
Current name looks like a callback rather than a enum, because "forward" is a verb.

Please consider changing the phrase "h2o_forward_informational" to "h2o_forward_informational_mode" or something (i.e. change the name to `h2o_forward_informational_mode_t`, `..._NONE` to `..._MODE_NONE`).